docs/source/backup/index.rst

```diff
@@ -1,272 +1,255 @@
+        This defaults to a subfolder of the backup folder but we **strongly suggest to back up to a different drive** (see note below).
 
-Before launching the script you need to create the file ``.pgpass`` to avoid having to enter your database password each time you use the script.
-It should look like (:download:`.pgpass <pgpass>`):
+    The configuration step creates two files in the "backup folder": a ``backup_info.json`` configuration file (can also be edited manually) and a ``start_backup.py`` script.
 
-.. literalinclude:: pgpass
+    .. note::
 
-where ``YOUR_DATABASE_PASSWORD`` is the password you set up for the database.
+        Using the same disk for your backup forgoes protection against the most common cause of data loss: disk failure.
+        One simple option is to use a destination folder mounted over ssh.
 
-.. note::
+        On Ubuntu, install ``sshfs`` using ``sudo apt-get install sshfs``.
+        Imagine you run your calculations on `server_1` and would like to back up regularly to `server_2`.
+        Mount a `server_2` directory on `server_1` using the following command on `server_1`:
 
-  Do not forget to put this file in ``~/`` (your home-directory) and to rename it ``.pgpass``.
-  Remember also to give it the right permissions (read and write): ``chmod u=rw .pgpass``.
+        .. code-block:: shell
 
-To dump the database in a file automatically everyday, you can add the following script :download:`backup-aiidadb-USERNAME <backup-aiidadb-USERNAME>` in the folder ``/etc/cron.daily/``, which will launch the previous script once per day:
+            sshfs -o idmap=user -o rw backup_user@server_2:/home/backup_user/backup_destination_dir/ /home/aiida_user/remote_backup_dir/
 
-.. literalinclude:: backup-aiidadb-USERNAME
-   :language: shell
+        Use ``gnome-session-properties`` in the terminal to add this line to the actions performed at start-up.
+        Do **not** add it to your shell's startup file (e.g. ``.bashrc``) or your computer will complain that the mount point is not empty whenever you open a new terminal...
 
-Where all instances of ``USERNAME`` must replaced by your actual user name.
-The ``su USERNAME`` makes the dumped file be owned by you rather than by ``root``.
-Remember to give the script the right permissions::
 
-  sudo chmod +x /etc/cron.daily/backup-aiidadb-USERNAME
+ 2. Run the ``start_backup.py`` script in the "backup folder" to start the backup.
 
-Finally make sure your database folder (``/home/USERNAME/.aiida/``) containing this dump file and the ``repository`` directory, is properly backed up by your backup software (under Ubuntu, 12.04: Backup -> check the "Folders" tab, 18.04: Backups -> check the "Folder to save" tab).
+    This will back up all data added after the ``oldest_object_backedup`` date.
+    It will only carry out a new backup every ``periodicity`` days, until a certain end date if specified (using ``end_date_of_backup`` or ``days_to_backup``).
```

Comment:
Since these terms have not yet been introduced, the place in the docs for their definitions should probably be mentioned, e.g.:
```suggestion
    It will only carry out a new backup every ``periodicity`` days, until a certain end date if specified (using ``end_date_of_backup`` or ``days_to_backup``), see :ref:`backup_configuration_options` below.
```
Also adding the tag `_backup_configuration_options` below, prior to the section "Configuration options".

Reply:
This goes for the previous line as well, although the reference to configuration options should probably only be done once.

Reply:
> Also adding the tag `_backup_configuration_options` below, prior to the section "Configuration options".

Did you miss this ^^^ ? :)

Reply:
thanks, yes I overlooked that this still needs to be added